.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# Code owners
+.github/CODEOWNERS @exercism/maintainers-admin
+
+# Synced files
+/global-files/ @exercism/maintainers-admin
+/tracks-files/ @exercism/maintainers-admin
```

Comment:
Why are there leading `/` but not above?

Reply:
Because this is a directory and the above is a file. At least that is how I interpreted it in: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax

Reply:
In v3 we used `reference/stories/**/*` for dirs fwiw.